sdk/keyvault/azure-keyvault-certificates/tests/test_examples_certificates_async.py

```diff
@@ -113,21 +110,21 @@ class TestExamplesKeyVault(AsyncKeyVaultTestCase):
     @AsyncVaultClientPreparer(enable_soft_delete=True)
     @AsyncKeyVaultTestCase.await_prepared_test
     async def test_example_certificate_list_operations(self, vault_client, **kwargs):
-        from azure.keyvault.certificates import CertificatePolicy, SecretContentType
+        from azure.keyvault.certificates import CertificatePolicy, SecretContentType, WellKnownIssuerNames
```

Comment:
Is there a reason not to place these imports at module level?

Reply:
no they're repetitive, moving them to the top while keeping some imports in the code samples. thanks!